src/index.html

```diff
@@ -3,11 +3,249 @@
+            anytime.
+          </p>
+        </div>
+
+
+        <div class="gallery__row gallery__row-2">
+          <img
+            class="gallery__photo gallery__photo-2"
+            src="/images/photos/image-3.png"
+            alt="bike frame 3"
+          >
+          <img
+            class="gallery__photo gallery__photo-1"
+            src="/images/photos/image-4.png"
+            alt="bike frame 4"
+          >
+        </div>
+        <div class="gallery__under">
+          <h3 class="gallery__feature gallery__feature-2">
+            Range & Integrated lights
+          </h3>
+          <p class="gallery__text">
+            The removable battery has up to 70km battery autonomy and weighs only 2.4 kg.<br> Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </div>
+
+
+        <div class="gallery__row gallery__row-3">
+          <img
+            class="gallery__photo gallery__photo-1"
+            src="/images/photos/image-5.png"
+            alt="bike frame 5"
+          >
+          <img
+            class="gallery__photo gallery__photo-2"
+            src="/images/photos/image-6.png"
+            alt="bike frame 6"
+          >
+        </div>
+        <div class="gallery__under">
+          <h3 class="gallery__feature gallery__feature-3">
+            Hydraulic disc brakes & Lightweight
+          </h3>
+          <p class="gallery__text">
+            Brakes with total stopping power the second you make contact. The removable<br> battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated<br> into the frame give you always-on visibility day and night.
+          </p>
+        </div>
+
+        <div class="gallery__button-container">
+          <a href="/" class="gallery__button">Explore</a>
+        </div>
+      </div>
+    </section>
+
+    <section class="contact">
+
+      <div class="contact__container">
+        <h2 class="contact__title">Contact us</h2>
+        <div class="contact__forms">
+          <div class="contact__inputs">
+
+            <label class="contact__field">
+              <input
+                type="name"
+                placeholder="Name"
+                class="contact__box"
+              >
+            </label>
+
+            <label class="contact__field">
+              <input
+                type="email"
+                placeholder="Email"
+                class="contact__box"
+              >
+            </label>
+
+            <label class="contact__field contact__field-last">
+              <textarea
+                class="contact__box
+                contact__box-textarea"
+                placeholder="Message"
+              >
+              </textarea>
+            </label>
+
+            <form
+              action="/"
+              method="post"
+            >
+              <button type="submit" class="contact__button">Send</button>
+            </form>
```

Comment:
oh no-no. Nest all input fields inside form - since form is not only the button, right?
to avoid an error, advise you to use
```
<form
  action="#"
  method="get"
>
```